platform/android/app/src/main/kotlin/coop/polypoly/polypod/network/Network.kt

```diff
@@ -124,7 +124,8 @@ open class Network(val context: Context) {
             connection.disconnect()
             return@withContext response
         }
-        return@withContext response
+        @Suppress("UNREACHABLE_CODE")
+        return@withContext response // unreachable code needed for return type inference
```

Comment:
I think that a better fix is to remove the return statement from the `finally` block. Same for the one below.